dev/how-to/migrate/from-mariadb-galera-cluster.md

```diff
@@ -0,0 +1,571 @@
+---
+title: Migrate Data from MariaDB Galera Cluster to TiDB
+summary: Learn to migrate data from MariaDB Galera Cluster to TiDB.
+category: how-to
+---
+
+# Migrate Data from MariaDB Galera Cluster to TiDB
+
+This document introduces how to migrate data from MariaDB Galera Cluster to TiDB. 
+
+## Hardware and system requirements
+
+### Hardware requirements
+
+- NIC bonding or teaming
+- PCIe SSDs (not SATA SSDs)
+- CPU power saving management disabled in BIOS
+- NUMA disabled in BIOS (`flush_caches` is optimized in Percona Server 5.7.10. For details, see [Improved NUMA support](https://www.percona.com/doc/percona-server/5.7/performance/innodb_numa_support.html))
+
+### Operating system requirements
+
+- Use CentOS 6.9 64-bit or CentOS 7.3-1611 64-bit as the operating system.
```

Comment:
Why do we refer to these specific release of CentOS 6 and CentOS 7? Surely it would be best if people used the most up-to-date releases of these OSes. (Current CentOS 7 is 7.6, for example.)